src/applications/mhv-supply-reordering/components/UnavailableSupplies.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { format } from 'date-fns';
+import PropTypes from 'prop-types';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+import { getUnavailableSupplies } from '../utilities/mdot';
+
+/**
+ * Shows a list of unavailable supplies if it exists.
+ * @param {*} mdotData the MDOT data for the user
+ */
+const UnavailableSupplies = mdotData => {
+  const unavailSupplies = getUnavailableSupplies(mdotData.supplies);
+  const cards = unavailSupplies
+    ?.sort((a, b) => a.productName.localeCompare(b.productName))
+    .map((supply, index) => (
+      <div key={`mhv-supply-unavail-${index}`}>
+        <va-card show-shadow>
+          <div>
+            <strong>{supply.productGroup}</strong> <br />
+            Device: {supply.productName} <br />
+            Quantity: {supply.quantity} <br />
+            Last ordered on{' '}
+            {format(new Date(supply.lastOrderDate), 'MMMM d, yyyy')}
+          </div>
+          {supply.availableForReorder && (
+            <p>
+              You can’t order this supply online until{' '}
+              {format(new Date(supply.nextAvailabilityDate), 'MMMM d, yyyy')}.
+              If you need this supply now call us at{' '}
+              <va-telephone contact="3032736200">303-273-6200</va-telephone> (
+              <va-telephone contact={CONTACTS['711']} tty />
+              ).
+            </p>
+          )}
+          {!supply.availableForReorder && (
+            <p>
+              This item is not available for reordering. To reorder, you can
+              call your VA healthcare team or send them a message.
+            </p>
+          )}
+        </va-card>
+        {index < unavailSupplies.length - 1 && <br />}
+      </div>
+    ));
+
+  return (
+    <div>
+      {unavailSupplies?.length > 0 && (
+        <div>
+          <h3>Unavailable for reorder</h3>
```

Comment:
Notice that ‘Need help?’ has a h2 tag, and ‘Unavailable for reorder’ has h3 tag - should they have the same heading level (eg. h2)?

Reply:
I'll be working on styling and matching Figma on subsequent PRs. Will make note of this.